src/pages/DonateWidget/Content.tsx

```diff
@@ -22,16 +19,11 @@ export default function Content({
   searchParams,
   classes = "",
 }: Props) {
-  const dispatch = useSetter();
-
-  useEffect(() => {
-    const donationRecipient: DonationRecipient = {
-      id: profile.id,
-      name: profile.name,
-      hide_bg_tip: !!profile.hide_bg_tip,
-    };
-    dispatch(setRecipient(donationRecipient));
```

Comment:
no more `"after-render"` via `dispatch + useEffect` setting of state, initial donation state is now set before render